src/Infrastructure/Handler/Export/BatchPropertyGroupExportCommandHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+
+namespace Ergonode\ExporterShopware6\Infrastructure\Handler\Export;
+
+use Ergonode\Channel\Domain\Repository\ChannelRepositoryInterface;
+use Ergonode\Channel\Domain\Entity\Export;
+use Ergonode\Channel\Domain\Repository\ExportRepositoryInterface;
+use Ergonode\ExporterShopware6\Domain\Command\Export\BatchPropertyGroupExportCommand;
+use Ergonode\ExporterShopware6\Domain\Entity\Shopware6Channel;
+use Ergonode\ExporterShopware6\Infrastructure\Processor\Process\PropertyGroupShopware6ExportProcess;
+use Ergonode\SharedKernel\Domain\Aggregate\AttributeId;
+use Webmozart\Assert\Assert;
+
+class BatchPropertyGroupExportCommandHandler
+{
+    private ExportRepositoryInterface $exportRepository;
+
+    private ChannelRepositoryInterface $channelRepository;
+
+    private PropertyGroupShopware6ExportProcess $process;
+
+    public function __construct(
+        ExportRepositoryInterface $exportRepository,
+        ChannelRepositoryInterface $channelRepository,
+        PropertyGroupShopware6ExportProcess $process
+    ) {
+        $this->exportRepository = $exportRepository;
+        $this->channelRepository = $channelRepository;
+        $this->process = $process;
+    }
+
+    public function __invoke(BatchPropertyGroupExportCommand $command): void
+    {
+        $export = $this->exportRepository->load($command->getExportId());
```

Comment:
Code the same as here BatchCustomFieldExportCommandHandler - maybe we can extract it some how?